assets/src/js/admin/onboarding-wizard/components/wizard/style.scss

```diff
@@ -2,6 +2,7 @@
 	min-height: 100%;
 	height: auto;
 	width: 100%;
+	overflow-x: hidden;
```

Comment:
@henryholtgeerts I think this needs to be set at a higher level to resolve the horizontal scroll bar issue.

Reply:
Looks like it needs to go at the `#onboarding-wizard-app` root.

Reply:
@kjohnson Thanks for pointing out where this should be addressed, was having difficulty replicating the horizontal scrolling issue in browsers locally! Believe that this commit should resolve the overflow issues: https://github.com/impress-org/givewp/commit/4221b69a0c4636ca88a339528dfb11730a267843